centaur/src/main/resources/standardTestCases/do_not_retry_rc0.test

```diff
@@ -0,0 +1,13 @@
+name: do_not_retry_rc0
+testFormat: workflowsuccess
+backends: [Papiv2]
+
+files {
+  workflow: retry_with_more_memory/do_not_retry_rc0.wdl
+  options: retry_with_more_memory/retry_with_more_memory.options
+}
+
+metadata {
+  workflowName: do_not_retry_rc0
+  status: Succeeded
```

Comment:
I could check the memory allocation here, but I could not figure out the JSON key syntax for when there is only a single run (as opposed to retry_same_memory_output_failure.imitate_oom_error.-1.1.runtimeAttributes.memory).

Reply:
You should just be able to remove the `.1` from the middle. If not, if you can paste a cut-down version of the actual Cromwell metadata for this task, it shouldn't be too difficult to reverse engineer what the path should be (🤞 )

Reply:
I did figure out the syntax, which maps to the format of the metadata but is different from how we check metadata when there are retries.